Dockerfile

```diff
@@ -260,19 +262,24 @@ USER root
 #
 ################################################################################
 
-ENV PIPENV_RUNTIME 2.7
+ENV PIPENV_RUNTIME 3.9
+
+USER root
+
+RUN update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.8 100 && \
```

Comment:
i don't think we need 3.8 here right?

Reply:
I added it as an alternative because it was also installed, but I don't think we really need this to be the case if we decide we want to use 3.9 - update-alternatives requires root to change the version used, so the only way someone could use python3.8 after these lines are run is by calling `python3.8` directly. I think it makes sense then to follow your suggestion and remove the line adding 3.8 as an alternative here.